src/timeman.cpp

```diff
@@ -94,17 +94,17 @@ void TimeManagement::init(Search::LimitsType& limits,
     if (limits.movestogo == 0)
     {
         // Use extra time with larger increments
-        double optExtra = std::clamp(1.0 + 12.5 * limits.inc[us] / limits.time[us], 1.0, 1.11);
+        double optExtra = limits.inc[us] < 500 ? 1.0 : 1.13;
```

Comment:
Did the tests use increments both below and above 500?

Reply:
The simplification was taken directly from PR #5076, which has passed tests at low time controls. Also there is a STC Elo estimate running, and if it's not largely impacted, then the simplification can be considered valid.

Reply:
I think maybe the most correct way of testing your patch would have been to test it against the simplification patch instead of master, since your patch also includes the simplification.

Reply:
fine with me as it is, could indeed have run it against fauzis patchs though